examples/step-22/doc/results.dox

```diff
@@ -727,12 +727,15 @@ because GMRES(k) scales worse with the problem size than CG, as we discussed
 above.  Nonetheless, the improvement by a factor of 3-6 for moderate problem
 sizes is quite impressive.
 
-<h5>Combining block preconditioner and multigrid</h5>
+
+<h5>Combining block the preconditioner and multigrid</h5>
```

Comment:
I am pretty sure you meant to write 'Combining the block preconditioner and multigrid'.